test/scripts/create-cluster.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Copyright 2019 The Kubeflow Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This shell script is used to build a cluster and create a namespace.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+CLUSTER_NAME="${CLUSTER_NAME}"
+ZONE="${GCP_ZONE}"
+PROJECT="${GCP_PROJECT}"
+NAMESPACE="${DEPLOY_NAMESPACE}"
+
+echo "Activating service-account ..."
+gcloud auth activate-service-account --key-file=${GOOGLE_APPLICATION_CREDENTIALS}
+
+echo "Creating cluster ${CLUSTER_NAME} ... "
+gcloud --project ${PROJECT} beta container clusters create ${CLUSTER_NAME} \
+    --addons=HorizontalPodAutoscaling,HttpLoadBalancing,Istio \
+    --zone ${ZONE} \
+    --cluster-version 1.12
```

Comment:
Let's go more modern if we can. 1.14?

Reply:
Tried 1.14 and last, get the error "error: SchemaError(io.k8s.api.rbac.v1beta1.Subject): invalid object doesn't have additional properties" when install knative, I do not know why. Let's keep 1.12 to workd around this.

Reply:
Seems 1.13 works. Let's use 1.1.3.